js/lazy-load-js.js

```diff
@@ -37,10 +39,27 @@
     if (Drupal.settings.js_defer[queueName].once) {
       return;
     }
+
+    // Make sure this last bit isn't run twice for the same queue
     Drupal.settings.js_defer[queueName].once = true;
-    LazyLoad.js(Drupal.settings.js_defer[queueName].scripts, function () {
-      Drupal.attachBehaviors(document, Drupal.settings);
-    });
+
+    // Run only new behaviors added from the queue, or load all, depending on
+    // queue's reattach_all_behaviors setting.
+    if (Drupal.settings.js_defer[queueName].reattach_all_behaviors === false) {
+      existingBehaviors = Object.keys(Drupal.behaviors);
```

Comment:
Not sure how much D7 users mind anymore, but Object.keys is relatively newish, not in IE8. You might instead, do a $.clone here, and check key exists instead of inArray.